setup.py

```diff
@@ -17,6 +17,7 @@
         'click==7.0',
         'requests>=2.22.0',
         'python-dotenv>=0.10.3',
+        'boto3',
```

Comment:
asi bych dal jako nepovinné

Reply:
To jsem si taky říkal, ale třeba ten gitpython je taky povinný.